### PR TITLE
Closes #223 Created ability to update price

### LINKED ISSUE #223: Create ability to update price in Manage Decks page
Price by default equals 0. Implement UI, DTO and REST endpoint.

### PR DESCRIPTION
**Pull request by OlehPaliuh**

### CODE REVIEW

File: src/app/page/cabinet/userDecks/user.decks.component.html [hunk 1]
```diff
@@ -123,13 +127,21 @@ <h4 class="modal-title">{{dialogName}}</h4>
...
+              </td>
+              <td>
+                <input type="number" [(ngModel)]="points"/>
+              </td>
+            </tr>
           </table>
         </div>
         <div class="modal-footer">
-          <button *ngIf="name && description && categoryId" class="btn btn-primary btn-pointer"
+          <button *ngIf="name && description && categoryId && points" class="btn btn-primary btn-pointer"
```
Comment (antonbakalets) [inline src/app/page/cabinet/userDecks/user.decks.component.html]: We can leave points as not required field to idicate that deck if free.

File: src/app/page/cabinet/userDecks/user.decks.service.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ export class UserDecksService {
 
   createDeck(deck: NewDeckDTO) {
     return this.orlp.post('api/categories/' + deck.categoryId + '/decks', deck)
-      .map((response: Response) => response.json());
+      .map((response: Response) => console.log(response.json()));
```
Comment (antonbakalets) [inline src/app/page/cabinet/userDecks/user.decks.service.ts]: Please no console.log as discussed before.

File: src/app/page/cabinet/userDecks/user.decks.component.html [hunk 3]
```diff
@@ -50,6 +50,10 @@
           <th>DeckSyntax</th>
           <td>{{selectedDeck.synthax}}</td>
         </tr>
+        <tr>
+          <th>DeckPrice</th>
```
Comment (antonbakalets) [inline src/app/page/cabinet/userDecks/user.decks.component.html]: The "space" character is missing here and there, but especially here in the user message.

**Review by antonbakalets — APPROVED** [review]
Please fix minor comments and consider as approved from my side.

**Review by hlibOstapenko — APPROVED** [review]

**Review by OleksandrBartosh — APPROVED** [review]

**Review by Maks-Chornyi — APPROVED** [review]

**Review by Maks-Chornyi — APPROVED** [review]

**Pull request merged**